### DISCUSSION TITLE
[Feature Request] Need Option to select or type only month and year in Date Components.

### DISCUSSION THREAD

**Discussion by gurlivleenkainth2000** [heroui-inc/heroui#4720] [Feature requests]
### Is your feature request related to a problem? Please describe.

Actually, in my project want to ask user only month and year only not date for fetching records from database for particular month of particular year but right now it not possible. 

### Describe the solution you'd like

Date Components should have all the native type which are related date and time inputs.

### Describe alternatives you've considered

For this usecase, i have to use native input field with type month and have to add some tailwindcss to outer element.
### CSS on input element
<img width="270" alt="image" src="https://github.com/user-attachments/assets/450d5f4c-4c0a-48eb-92ae-099fd6051543">

### No Classes
<img width="272" alt="image" src="https://github.com/user-attachments/assets/0ecbfe0f-b848-4033-9287-5623d05f972c">

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1520/[feature-request]-need-option-to-select-or-type-only-month-and-year-in">ENG-1520 [Feature Request] Need Option to select or type only month and year in Date Components.</a></p>
